pkg/quarterdeck/api/v1/client.go

```diff
@@ -485,6 +489,21 @@ func (s *APIv1) InviteCreate(ctx context.Context, in *UserInviteRequest) (out *U
 	return out, nil
 }
 
+func (s *APIv1) InviteAccept(ctx context.Context, token string) (out *LoginReply, err error) {
+	endpoint := fmt.Sprintf("/v1/invites/%s", token)
```

Comment:
It's not typical that you POST `nil` data to an endpoint; the token is data not an identity. Is it possible to do `POST /v1/invites` and put the token in the body?

Reply:
Or alternatively do a POST to `/v1/invites/:userID`? The client could then parse the userID out of the token.